test/behaviors/conference.behavior.js

```diff
@@ -533,6 +532,121 @@ function shouldBehaveLikeConference () {
+          await register({conference, deposit, user:accounts[10 + i], owner})
+      } 
+
+      // [ 110 ], accounts[11], accounts[12]
+      const maps = [ toBN(0).bincn(1).bincn(2) ];
+
+      await conference.finalize(maps, {from:owner});
+      await conference.ended().should.eventually.eq(true)
+      
+      payoutAmount = new EthVal(await conference.payoutAmount());
+      clearFee = await conference.clearFee();
+      fees = payoutAmount.mul(clearFee).div(1000).toString(10);
+
+      assertBalanceWithDeposit((await getBalance(conference.address)), mulBN(deposit, 4))
+    })
+
+    it('fees calculation works correctly', async function(){
+      let previousBalanceOne = await getBalance(accounts[11]);
+      let previousBalanceTwo = await getBalance(accounts[12]);
+      let previousBalanceOwner = await getBalance(owner);
+
+      await wait(20, 1);
+      await conference.clearAndSend({from:owner});
+
+      let diffOne = new EthVal(await getBalance(accounts[11])).sub(previousBalanceOne)
+      let diffTwo = new EthVal(await getBalance(accounts[12])).sub(previousBalanceTwo)
+      let diffOwner = new EthVal(await getBalance(owner)).sub(previousBalanceOwner)
+
+      assert.isOk(diffOne.eq(payoutAmount.sub(fees)))
+      assert.isOk(diffTwo.eq(payoutAmount.sub(fees)))
+      assert.isOk(diffOwner.gt(new EthVal(fees).mul(1.9)))
+
+      assertBalanceWithDeposit((await getBalance(conference.address)), 0)
+    })
+
+    it('calling with a number of attenders', async function() {
+      let previousBalanceOne = await getBalance(accounts[11]);
+      let previousBalanceTwo = await getBalance(accounts[12]);
+
+      await wait(20, 1);
+      await conference.clearAndSend(1);
+      
+      let diffOne = new EthVal(await getBalance(accounts[11])).sub(previousBalanceOne)
+      let diffTwo = new EthVal(await getBalance(accounts[12])).sub(previousBalanceTwo)
+      
+      assert.isOk(diffOne.eq(payoutAmount.sub(fees)))
+      assert.isOk(diffTwo.lt(payoutAmount.sub(fees)))
+
+      assertBalanceWithDeposit((await getBalance(conference.address)), mulBN(payoutAmount, 1))
+    })
+
+    it('calling with a number of attendees greater than the contract\'s one', async function() {
+      await wait(20, 1);
+      await conference.clearAndSend(5).should.be.rejected;
+
+      assertBalanceWithDeposit((await getBalance(conference.address)), mulBN(payoutAmount, 2))
+    })
+
+    it('consecutive calls', async function() {
+      let previousBalanceOne = await getBalance(accounts[11]);
+      let previousBalanceTwo = await getBalance(accounts[12]);
+
+      await wait(20, 1);
+      await conference.clearAndSend(1);
+      await conference.clearAndSend(1);
+      
+      let diffOne = new EthVal(await getBalance(accounts[11])).sub(previousBalanceOne)
+      let diffTwo = new EthVal(await getBalance(accounts[12])).sub(previousBalanceTwo)
+      
+      assert.isOk(diffOne.eq(payoutAmount.sub(fees)))
+      assert.isOk(diffTwo.eq(payoutAmount.sub(fees)))
+
```

Comment:
Can you call `await conference.clearAndSend(1);` one more time here to check that it fails because there are no left over? This is slightly different to you `calling with a number of attendees greater than the contract` test.

Reply:
Done in the last commit!